File: MCR PROJECT/Assets/Script/Requetes/PayementSalaire.cs
/** @author Max
*/



namespace MODEL{
	class PayementSalaire : Requete {

		
		public override bool doitTraiter(Tresorier t) {
			return true;
		}

		public override bool doitTraiter(Tamponeur t) {
			return true;
		}
		
		public override bool doitTraiter(Chef c) {
			return true;
		}
	}
}

File: Java/Requetes/PayementSalaire.java
/** @author Max
*/




public class PayementSalaire extends Requete {

	@Override
	public bool doitTraiter(Tresorier t) {
		return true;
	}

	@Override
	public bool doitTraiter(Tamponeur t) {
		return true;
	}

	@Override
	public bool doitTraiter(Chef c) {
		return true;
	}
}
